src/example.test.js

```diff
@@ -0,0 +1,28 @@
+var tester = require('ts-starter');
+var goTo = tester.GoTo;
+var isEqual = tester.IsEqual;
+var resizeViewport = tester.ResizeViewport;
+tester.registerAction(goTo);
+tester.registerAction(isEqual);
+tester.registerAction(resizeViewport);
+
+tester.addTest('simple test', [
+    {
+        title: 'goes to google.com',
+        action: goTo,
+        args: ['http://www.google.com'], // the url to go to
+        outs: ['$a'] // a page instance
```

Comment:
We don't need this output if we use the same `page`.

Reply:
Yep, done